.github/workflows/docker-image.yml

```
@@ -73,10 +73,11 @@ jobs:
           username: ${{ secrets.DOCKER_HUB_USERNAME }}
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
 
-      - name: Build and push Docker image
+      - name: Build and push Docker image (Docker Hub)
         uses: docker/build-push-action@v4
```

actionlint (1.7.8)

the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)